recipe/meta.yaml

```diff
@@ -1,52 +1,104 @@
-  patches:
-    # https://patchwork.freedesktop.org/patch/214086/
-    - 1-1-gallivm-Fix-build-after-llvm-r328839.patch
+  fn: mesa-{{ version }}.tar.xz
+  url: ftp://ftp.freedesktop.org/pub/mesa/mesa-{{ version }}.tar.xz
+  sha256: 8e49585fb760d973723dab6435d0c86f7849b8305b1e6d99f475138d896bacbb
 
 build:
   skip: True  # [win]
-  number: 1
-  # Do we still need this???? - hmaarrfk - 18.0.0 -- April 2020
+  number: 0
+
   track_features:
-      - mesalib
+    - mesalib
 
 requirements:
   build:
-      - {{ compiler('c') }}
-      - {{ compiler('cxx') }}
-      - pkg-config
-      # still too hard to get rid of python2
-      - python 2
-      - mako
-      - zlib
-      - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('xorg-x11-proto-devel') }}
+    - {{ cdt('libxxf86vm-devel') }}
+    - {{ cdt('libxshmfence-devel') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - meson
+    - mako
+    - flex
+    - bison
+    - pkg-config
+
   host:
-      - pthread-stubs
-      - llvmdev
-      - expat
-      - zlib
+    - expat
+    - zlib
+    - libdrm                       # [linux]
+    - elfutils                     # [linux]
+    - ncurses
+    - llvmdev
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-xf86vidmodeproto
+    - xorg-libxrandr
+    - xorg-libxfixes
+    - xorg-glproto
+    - libxcb
+    - xorg-libxdamage
+
+  run:
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-xf86vidmodeproto
+    - xorg-libxrandr
+    - xorg-libxfixes
+    - xorg-glproto
+    - libxcb
+    - xorg-libxdamage
+
   run_constrained:
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
 
 test:
+  requires:
+    - pkg-config
+
   commands:
-      - test -f $PREFIX/lib/libOSMesa32${SHLIB_EXT}  # [unix]
+    - test -f $PREFIX/lib/dri/kms_swrast_dri${SHLIB_EXT}  # [linux]
+    - test -f $PREFIX/lib/dri/swrast_dri${SHLIB_EXT}      # [linux]
+    - test -f $PREFIX/lib/libgbm${SHLIB_EXT}              # [linux]
+    - test -f $PREFIX/lib/libGLESv1_CM${SHLIB_EXT}        # [linux]
+    - test -f $PREFIX/lib/libGLESv2${SHLIB_EXT}           # [linux]
+    - test -f $PREFIX/lib/libEGL${SHLIB_EXT}              # [linux]
+    - test -f $PREFIX/lib/libGL${SHLIB_EXT}               # [linux]
+    - test -f $PREFIX/lib/libOSMesa${SHLIB_EXT}           # [linux]
+    - test -f $PREFIX/lib/libglapi${SHLIB_EXT}            # [linux]
+
+    - test -f $PREFIX/lib/libGLESv1_CM.1${SHLIB_EXT}  # [osx]
+    - test -f $PREFIX/lib/libGLESv2.2${SHLIB_EXT}     # [osx]
+    - test -f $PREFIX/lib/libOSMesa.8${SHLIB_EXT}     # [osx]
+    - test -f $PREFIX/lib/libGL.1${SHLIB_EXT}         # [osx]
+    - test -f $PREFIX/lib/libglapi.0${SHLIB_EXT}      # [osx]
+
+    - pkg-config --exists glesv1_cm
+    - pkg-config --exists glesv2
+    - pkg-config --exists osmesa
+    # Disable gl, egl for linux, issue with xxf86vm missing dependency, disbale OpenGL?
```

Comment:
What does this mean?

Reply:
I don't know, it is from #33 ;-)

Reply:
Should we add a new dependency? Or do yo uwant to leave this feature as one that we should include in a future build?

Reply:
I don't know. After checking again, these lines seem to come from #30. Maybe @wolfv knows more?